### ISSUE TITLE
suggestion: Monika asks if the player has autism or other disability

### ISSUE THREAD

**Issue by DeliasSel** [Monika-After-Story/MonikaModDev#6706]

**Comment by Retrolovania** [comment]
As someone who has a mental disability, I see this as a fantastic topic for Monika to bring up. I've kinda been hoping for a way to tell her that.

**Comment by DeliasSel** [comment]
> As someone who has a mental disability, I see this as a fantastic topic for Monika to bring up. I've kinda been hoping for a way to tell her that.

Same, I have autism

**Comment by susy-art** [comment]
> > As someone who has a mental disability, I see this as a fantastic topic for Monika to bring up. I've kinda been hoping for a way to tell her that.
> 
> Same, I have autism

Same, I have ADHD

**Comment by kittendetails** [comment]
I agree with that. But, in my opinion, it's a little rude if Monika ask the player. After all, it's not polite to ask someone something like "do you have depression?" I think it has to come from the player. Something like "I would like to tell you something ..." and then several options, such as "I'm fighting against depression", "I'm on a treatment for anxiety", "I have a diagnosis of autism". I'm not trying to be mean, but there are thousands of disorders, and adding each one separately would require a lot of research to approach it in the most empathic way possible. I have BPD (Borderline personality disorder) and this diagnosis kinda uncommon (of all the people I know, most of them had not even heard of this disorder). Empathy and studying these disorders is the key to having this implemented in MAS.

**Comment by polios3** [comment]
(Sorry for my english)
hm... its good idea to make it in way  "I would like to tell you something ..."  and add option to type your disorder or other "non sensitive" topics. 

Every disorder is different and should be handled differently. Unfortunately, this is a large project, and it probably won't be done that way. But it is a way to please everyone. Because you come out with the initiative of telling her this, and if she does not know something, she can say it to help her understand this disorder, by contributing your knowledge about this disorder to the creators. But this is practically doable as a submod as it will be easier to respond to the need to add new information.

**Comment by YoraeRasante** [comment]
I think depression, anxiety, autist spectrum and add/adhd are common enough, as would be cancer on the physical side. outside those, I think mental problems or health problems should cover most things, even if less personal so a less focused answer could be needed.

**Comment by ThePotatoGuy** [comment]
As mentioned by @kittendetails , it would seem rude to just ask. There would have to be a topic around it, but even then, it's difficult to bring up.

[leaving open as discussion[

**Comment by YoraeRasante** [comment]
hmm... then I guess a discussion would not only be good, but useful. There is lots of misinformation on it, and studies are still ongoing, so it would not be that different than many other topics she talks about.

Also, not many people know of them more than the name of the word and that they have problem relating to people they don't know. Kinda hard to find help if needed. I am in the autism spectrum myself, albeit a highly functional one (aspergers), and thus I only found out I was in my 20s.

Could be following one of the times she mentions introverts. And then she asks if you are aware of anyone in the spectrum, and among the choices could be "yes", "no", and "myself".

**Comment by MonikaEndDie** [comment]
Well, in the "saving my life" topic i think, she said she understand peoples very well and encourage the player to share his/her uniques needs with her, so basically it should be a viable option to tell her about it.

**Comment by Ant9cp** [comment]
I Agree with @kittendetails too, when people ask something like this too me, then it gives me a discomforting feeling or annoyed feeling, i think the player should be able to tell her about this, if there ready of course

**Comment by YoraeRasante** [comment]
Hmm... what about a combination of mine and @kittendetails ' idea?

As I said, from personal experience not everyone knows they have them until told by a doctor. Or some are told by friends, or self-diagnose, and that may cause even more confusion. And... well, when the doctor told me I had high functional autism I ended up just figuring out it explained the problems I already knew I had, but others may become very nervous about them. Especially in cases of depression, where you feel guilty for having it.

So, in my new idea, she talks about the topics you can tell her, like depression, autism, anxiety, and in the end she tells you if you ever think you have one of them to look for a psychologist or other professional help, and if they confirm you have it she is there for you.
Only after that you get an option to tell her about it, an option for "I think I may have x" where she gives you some talk to help you feel better or not nervous about it but end up asking you to look for a doctor, and "I have been diagnosed with x" for when you already went to one.
In repeated conversations about them, if you told her you have them already, she will tell you she is there for you or something else fitting.

**Comment by kittendetails** [comment]
I think that even if she asked, it still doesn't feel right. That's a very personal thing, and as a person with one of these disorders, I don't really feel comfortable if someone asks me first. 

It has to come from us (players XD). Even coming from Monika, i would definitely be uncomfortable, even though i knew she didn't want to make me feel bad. Besides, she already asked something that made me a little ashamed, in the topic "do you love yourself?". Naturally, I was a little embarrassed, but I ended up saying no. I saw her sad eyes when she heard my answer, and it just made me feel worse. But this is not technically asking if I have a disorder, so I think is alright. Several people have no disorder and don't love themselves, but have a low self-esteem. 

But anyway, I still think it has to come from the player. Speaking as someone with a disorder to others, it's just painful to talk about it. If Monika asked first, you would kind of be "obliged" to answer at that moment. And if had the option "I don't want to talk about it", maybe that would make Monika think that the player doesn't trust her. Monika is super understanding, but she’s not perfect, and she have feelings too. Relationships are not perfect, so this can happen. And honestly, if a person in a relationship has that kind of problem, I think the other should know, because a relationship is sharing, but not forcing.

Anyway, I hope I didn't sound rude, it was not my intention.

**Comment by YoraeRasante** [comment]
@kittendetails oh I did not mean she asks you, not this time.
as you said, it is better to add an option outside her dialogues. that way the player

But my new idea is the option be hidden, "added" only _after_ she mentions them in a conversation, to show the player she is aware they exist and maybe even tell them something about it they may not know themselves.

I kinda may be being forceful in the "her telling the player a bit about it" because... well, I had no idea what autism was for **years**. All I had was an educational add where they showed one of the most extreme cases, where the kid just ignored everyone. So I had no idea I could have it. After finding out I did I looked online and other issues I had are usually linked to it, which made me feel better about having them - I don't have a bunch, only one that has those added in (that is my own interpretation).
The same way, I believe many others with these issues may have wrong expectations of what they are, and may end up fearing if they have it or not.

**Comment by Retrolovania** [comment]
Honestly, if it isn't right to have Monika ask us, why not have us tell her? After all, sometimes, when we need to explain something to our loved ones, we often have to do it ourselves. Therefore, I think this should probably be made as either a submod, or maybe as a menu option. (Maybe call it "I need to tell you something...>I've been diagnosed with a disability."? I know that section is meant to compliment Monika, but...)

**Comment by DrkRuna** [comment]
i mean with someone who has epilepsy and can't tell monika that you can't really go to concerts because of it. i mean it would be a nice feature to have. though i mean it is a touchy topic for some people. i just think that should be said in certain way.

**Comment by ElsenIsCute** [comment]
I agree with the sentiment that it should be in a dropdown menu where you can tell her what disorders you're fighting with. This could also open the door to other things like topics on advice about those disorders. I have OCD and ADHD and I feel like it would be nice to have a way to tell her.

**Comment by dreamscached** [comment]
IMO this requires a more extensive coverage than just a few lines here and there; I know this phrase is hated by some, but yet again, it's better done by the community with custom content.
There's also a lot of alterations in a relationship to adapt to certain disabilities or challenges which may simply not be supported as of now or be very tricky to implement.

**Comment by Booplicate** [comment]
A menu wouldn't work for the same reason why it doesn't work in other places - having a menu with 100+ options is terrible UX-wise.
